### ISSUE TITLE
Longman pronunciations

### ISSUE THREAD

**Issue by AizenAnki** [AwesomeTTS/awesometts-anki-addon#169]
Dear developers of AwesomeTTS,

I love your add-on.

I wanted to ask you if it were possible to add also British English and American English pronunciations from the Longman dictionary (https://www.ldoceonline.com/), as you already do for the Cambridge and Oxford ones.

Although it is true that these already provide the 2 pronunciations, I believe it is better to have (also) the Longman ones. Here is the rationale: the best Pronunciation Dictionary of English in circulation is the Longman's one. There may be some debate about it, but it is nowadays undebated that its curator (Prof. John Wells) is the foremost authority in the field. Pronunciation dictionaries have some differences between them: this is normally limited to a certain amount of words, but there are.

The point is, that the recordings used in the Pronunciation Dictionary, and the ones provided in the Longman Online Dictionary are the same (I checked this by inspection). True, the Pronunciation Dictionary contains several more words (i.e. proper names, cities, ...), but for learners of English, the most common words are the most important ones, and they are available in the Online Dictionary.

Long story short, for people learning also with the Longman Pronunciation Dictionary, the addition of the Longman’s recordings would be even more helpful.

**Comment by lucwastiaux** [comment]
It's possible but of course this would only work for decks which are made up of vocabulary only, without any sentences. In my case, a huge majority of my decks are full sentences, so retrieving pronunciations from a dictionary wouldn't work.

**Comment by Sowa0** [comment]
I agree, it would be great to add audio from the Longman Dictionary of English (LDCOE). Simple html web scrapping works as the url of the mp3 is available in the html.

`<span data-src-mp3="https://www.ldoceonline.com/media/english/breProns/contemporary_n0205.mp3?version=1.2.30" class="speaker brefile fas fa-volume-up hideOnAmp" title="Play British pronunciation of   ##contemporary">&nbsp;</span>`

`<span data-src-mp3="https://www.ldoceonline.com/media/english/ameProns/contemporary1.mp3?version=1.2.30" class="speaker amefile fas fa-volume-up hideOnAmp" title="Play American pronunciation of contemporary">&nbsp;</span>`

**Comment by luc-vocab** [comment]
are yoeu still interested in this Longman dictionary if I implement a service for it in HyperTTS ?

**Comment by Sowa0** [comment]
I personally do not need it and use HyperTTS for other languages but I think it would be good to offer that option since Longman is one of the major dictionaries used by English learners
